lib/takeaway.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "dish"
+require_relative "order"
+require_relative "menu"
+
+class Takeaway
+
+  attr_reader :order, :total, :menu
+
+  def initialize(order = Order.new, menu = Menu.new )
+    @order = order
+    @menu = menu
+    @total = 0
+  end
+
+  def see_menu
+    menu.menu
+  end
+
+  def add(dish, quantity = 1)
```

Comment:
The order already has this functionality - there's no need to repeat it in Takeaway